packages/mystjs/package.json

```diff
@@ -33,11 +33,12 @@
     "test:watch": "jest --watchAll",
     "lint": "eslint \"src/**/*.ts\" -c .eslintrc.js",
     "lint:format": "prettier --check src/*.ts src/**/*.ts",
-    "build:bundle": "esbuild src/index.ts --bundle --outfile=dist/index.umd.min.js --platform=browser",
+    "build:bundle:cjs": "esbuild src/index.ts --bundle --outfile=dist/index.cjs.js --platform=node",
+    "build:bundle:browser": "esbuild src/index.ts --bundle --outfile=dist/index.umd.min.js --platform=browser --minify",
```

Comment:
This is the main change, I have added a CJS dependency, as well as actually minified the umd build. There likely needs to be more testing on the mystjs for the browser as I am not convinced that it will work, but this can go in now as it is an improvement.